motor/motor_control/launch/keyboard_teleop.launch

```diff
@@ -0,0 +1,7 @@
+<launch>
+  <!-- Launch keyboard teleop node -->
+  <node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py">
+    <!-- Remap cmd_vel topic name from keyboard teleop -->
+    <remap from="cmd_vel" to="cmd_vel"/>
```

Comment:
Isn't this a bug ? You should remap cmd_vel to key_vel right ?

Reply:
Deprecated, this isn't the launch file we use for teleop. Removing from branch for clarity